core/src/main/kotlin/org/evomaster/core/output/TestSuiteSplitter.kt

```diff
@@ -19,6 +21,38 @@ object TestSuiteSplitter {
 
         return when(type){
             EMConfig.TestSuiteSplitType.NONE -> listOf(solution)
+            EMConfig.TestSuiteSplitType.CODE -> splitByCode(solution)
         }
     }
+
+    fun <T:Individual> splitByCode(solution: Solution<T>): List<Solution<T>>{
+
+
+        val s500 = solution.individuals.filter {
+            it.evaluatedActions().any { ac ->
+                (ac.result as RestCallResult).getStatusCode() == 500
+
```

Comment:
clarify in a comment that here we just look at 500, and not at all the 5xx (eg, 501), as those are not so interesting (eg, not really related to find bugs)